test/integration/narrow/routes/capture.test.js

```diff
@@ -0,0 +1,81 @@
+describe('Capture test', () => {
+  jest.mock('ffc-messaging')
+  jest.mock('../../../../app/plugins/crumb')
+  const createServer = require('../../../../app/server')
+  let server
+  const url = '/capture'
+
+  jest.mock('../../../../app/debt')
+  const { getDebts } = require('../../../../app/debt')
+
+  const debts = [{
+    scheme: 'SFI Pilot',
+    frn: '1234567890',
+    reference: 'SFIP1234567',
+    netValue: 1000.00,
+    debtType: 'Irregular',
+    recoveryDate: '19/01/2022',
+    attachedDate: '',
+    createdBy: 'John Watson'
+  },
+  {
+    scheme: 'SFI',
+    frn: '1234567891',
+    reference: 'SFIP1234568',
+    netValue: 570.00,
+    debtType: 'Administrative',
+    recoveryDate: '18/01/2022',
+    attachedDate: '18/01/2022',
+    createdBy: 'Steve Dickinson'
+  }]
+
+  beforeEach(async () => {
+    getDebts.mockResolvedValue(debts)
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterEach(async () => {
+    jest.resetAllMocks()
+    await server.stop()
+  })
+
+  describe('GET requests', () => {
+    const method = 'GET'
+
+    test('GET /capture route returns 200', async () => {
+      const options = {
+        method,
+        url
+      }
+
+      const response = await server.inject(options)
+      expect(response.statusCode).toBe(200)
+      expect(response.request.response.variety).toBe('view')
```

Comment:
I would encourage these to be broken up into tests for:
- 200 code
- view returned
- view type to be `capture`

Reply:
The 3rd point is missing

Reply:
These are all part of the response and splitting up my be over kill.